dev/PostgresDockerfile

```diff
@@ -0,0 +1,4 @@
+FROM postgres:11.5
```

Comment:
Could you create `db` folder in the root folder and move this file and `initdb.sql` into `db` folder?

Reply:
Ok, I will do it